automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -0,0 +1,185 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, Config}
+
+import scala.concurrent.duration._
+
+
+trait Orchestration extends FireCloudClient with LazyLogging {
+
+  def responseAsList(response: String): List[Map[String, Object]] = {
+    mapper.readValue(response, classOf[List[Map[String, Object]]])
+  }
+
+  private def retry[T](remainingBackOffIntervals: Seq[FiniteDuration])(op: => () => Boolean): Boolean = {
```

Comment:
Also `op: => () => Boolean` seems wrong. There are too many layers of indirection. This is a "by-name parameter which returns a function which takes no arguments and returns a Boolean". But all you need is a "by-name parameter which returns a Boolean". So:
```
private def retry[T](remainingBackOffIntervals: Seq[FiniteDuration])(op: => Boolean): Boolean = {
  op match {
    // snip
  }
}
```
This should give you the behavior you need. More info: https://tpolecat.github.io/2014/06/26/call-by-name.html